BE/OwnershipAndControl/Executives.rdf

```diff
@@ -177,18 +233,13 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>board capacity</rdfs:label>
-		<skos:definition>the capacity that resides in the board of directors of a company</skos:definition>
+		<skos:definition>authority to act in a fiduciary capacity with respect to the organization, including but not limited to determining and executing corporate policy</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-exec;BoardMember">
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;AuthorizedParty"/>
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;LegallyDelegatedAuthority"/>
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;OrganizationMember"/>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
-				<owl:allValuesFrom rdf:resource="&fibo-be-le-lp;LegalPerson"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
```

Comment:
This restriction on isAPartyTo is a bit pointless, since it does not constrain to the Agreement of the same company. In general I think it's safe to assume that any board member is party to such an agreement - it's not FIBO's job to track each individual who actually signed!
Likewise the restriction on isPlayedBy has a lot of complexity and still does not constrain membership to the board of the same company.
In summary I don't think we need wither - just a reified relationship (giving their term etc. ) from Board to LegalPerson